packages/server/src/integrations/tests/utils/index.ts

```diff
@@ -14,4 +15,4 @@ export interface DatabaseProvider {
   datasource(): Promise<Datasource>
 }
 
-export const databaseTestProviders = { postgres, mongodb, mysql }
+export const databaseTestProviders = { postgres, mongodb, mysql, s3 }
```

Comment:
The `s3` provider doesn't _quite_ fit the interface expected by `DatabaseProvider` above, as it doesn't have a `datasource(): Promise<Datasource>` function. It may be better to split the `s3` provider out into its own `objectStoreTestProviders` package.

Reply:
Sorted, with the move to minio image we now call like this`objectStoreTestProviders.minio.start()`